frontend/src/metabase/writeback/containers/ActionParametersInputModal.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import Modal from "metabase/components/Modal";
+import ModalContent from "metabase/components/ModalContent";
+
+interface Props {
+  onClose: () => void;
+  title: string;
+  children: React.ReactElement;
+}
+
+export default function ActionParametersInputModal({
+  onClose,
+  title,
+  children,
+}: Props) {
+  return (
+    <Modal onClose={onClose}>
+      <ModalContent title={title} onClose={onClose}>
+        {children}
+      </ModalContent>
+    </Modal>
+  );
+}
```

Comment:
Seems like this component doesn't do a lot of work. Maybe it'd make sense to render `ActionParametersInputForm` inside of it instead of just rendering whatever is passed as `children`? Or maybe just inline these `Modal` and `ModalContent` right into the `ActionForm` component